Blog: PostHog vs Plausible (#9514)
PostHog/posthog.com: contents/blog/posthog-vs-plausible.mdx

---
title: "In-depth: PostHog vs Plausible"
date: 2024-10-01
author:
 - ian-vanagas
rootpage: /blog
featuredImage: >-

>   https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-marketing/marketing-hog.jpg

Reviewer: Let's get custom art for this is you haven't requested it already

featuredImageType: full
category: General
tags:
  - Comparisons
---

import { ComparisonTable } from 'components/ComparisonTable'
import { ComparisonHeader } from 'components/ComparisonTable/header'
import { ComparisonRow } from 'components/ComparisonTable/row'

## How is PostHog different?

### 1. We're free (forever)

Our ["ridiculously cheap" tier](/pricing) includes all of the features of PostHog for free with an allowance of 1M events per month. For Plausible, this would cost a minimum of $69/m. 

> Beyond PostHog's free tier, we try to be as cheap as possible and transparent. We can do this because we're efficient. We don't splurge on massive sales teams, or D-list comedians to host an annual convention you'll never attend.

Reviewer: I don't think this line works in this context because Plausible wouldn't do these things, either.

Want to know how much we'll charge? See our [pricing calculator](/pricing).

### 2. We're more than just web analytics

> Although [web analytics](/web-analytics) is a big part of PostHog, it's just a part of our overall platform. We have many more tools to help you analyze, test, observe, and deploy your site and app.

Reviewer: ```suggestion 
[Web analytics](/web-analytics) is a core feature of PostHog, but we offer many more tools to help you analyze, test, observe, and deploy your site and app.
```

> These include [product analytics](/product-analytics), [session replay](/session-replay), [surveys](/surveys), [A/B testing](/experiments),  [feature flags](/feature-flags), a [data warehouse](/data-warehouse), and more 

Reviewer: ```suggestion
These include [product analytics](/product-analytics), [session replay](/session-replay), [surveys](/surveys), [Experiments](/experiments),  [feature flags](/feature-flags), a [data warehouse](/data-warehouse), and more 
```

### 3. We're multi-platform

> Plausible is entirely focused on the web. PostHog has a major focus on the web, but also other platforms like mobile, backend, and [API](/docs/api).

Reviewer: ```suggestion
Plausible is entirely focused on the web, whereas PostHog supports both web tracking and other platforms, like mobile, backend, and [API](/docs/api).
```

This isn't limited to event capture and features either. Our [data warehouse](/data-warehouse) lets you import and query data from sources like [Stripe](/docs/data-warehouse/setup/stripe), [Hubspot](/docs/data-warehouse/setup/hubspot), and [Postgres](/docs/data-warehouse/setup/postgres), and our data pipelines let you send data to destinations like [Slack](/docs/cdp/destinations/slack), [S3](/docs/cdp/batch-exports/s3), or a custom [HTTP webhook](/docs/cdp/destinations/webhook).

## Web analytics

> Both Plausible and PostHog are [Google Analytics alternatives](/blog/ga4-alternatives). This means a big focus on tracking and analyzing your website. 

Reviewer: ```suggestion
Both Plausible and PostHog are [Google Analytics alternatives](/blog/ga4-alternatives), so they're ideal tracking and analyzing your website. 
```

  <ComparisonRow column1={true} column2={true} feature="Sessions" description="Track unique sessions and their durations" />
  <ComparisonRow column1={true} column2={true} feature="Traffic breakdown" description="See where your visitors and conversions are coming from" />
  <ComparisonRow column1={true} column2={true} feature="Conversions" description="Track actions you want users to take" />
  <ComparisonRow column1={true} column2={true} feature="Bounce rate" description="See the percentage of users that leave after one pageview" />
  <ComparisonRow column1={true} column2={true} feature="Entry and exit paths" description="See the pages users first visit and the last ones before they leave" />
  <ComparisonRow column1={true} column2={true} feature="Outbound clicks" description="See the links that take users away" />
  <ComparisonRow column1={true} column2={true} feature="UTM tracking" description="Track marketing campaigns with UTM tags" />
  <ComparisonRow column1={true} column2={true} feature="Cookieless tracking" description="Track users without cookies" />
  <ComparisonRow column1={true} column2={true} feature="Snippet install" description="Install HTML snippet" />

> </ComparisonTable>

Reviewer: Is there anything Plausible has/does that we don't?

## Product analytics

> For more customization, both in capturing data and analyzing it, teams rely on [product analytics](/product-analytics). Although not a focus of Plausible, it does have some product analytics features, but not nearly as many as PostHog's full suite. 

Reviewer: ```suggestion
Teams rely on [product analytics](/product-analytics) when they need more customization in both capturing and analyzing date. Although not a focus of Plausible, it does have offer some basic product analytics features for those that need them.
```

I think we can let the table do the talking here.

> 
> Many of Plausible's product analytics features, like custom properties and [funnels](/docs/product-analytics/funnels), are on their more expensive "Business" plan. 

Reviewer: ```suggestion
```

  <ComparisonRow column1={true} column2="Limited" feature="Funnels" description="Track users through a sequence of events" />
  <ComparisonRow column1={true} column2={false} feature="Retention" description="Visualize which users stay, for how long" />
  <ComparisonRow column1={true} column2={false} feature="Custom dashboards" description="Build a custom dashboard of your own or use a template" />
  <ComparisonRow column1={true} column2={false} feature="Cohorts" description="Combine users based on properties and events for group analysis" />
  <ComparisonRow column1={true} column2={false} feature="User profiles" description="View user demographic data such as country, language, and device type" />
  <ComparisonRow column1={true} column2={false} feature="Group analytics" description="Track metrics at a company and account level" />
  <ComparisonRow column1={true} column2={false} feature="Paths" description="Track user flows and where they drop off" />
  <ComparisonRow column1={true} column2={false} feature="SQL" description="Query your data with SQL" />
</ComparisonTable>

> 

Reviewer: ```suggestion

> 💡 **Good to know:** Many of Plausible's product analytics features, like custom properties and [funnels](/docs/product-analytics/funnels), are on its more expensive "Business" plan. It's roughly twice the price of the Growth plan and includes up to 10 team members and priority support.

```

> ## Platform

Reviewer: We should add something around team member limits here. Plausible has them, we don't.

Beyond web analytics is where PostHog really shines. We provide all of the tools developers need to build a successful product (and they all work together). 
<ComparisonTable column1="PostHog" column2="Plausible">
  <ComparisonRow column1={true} column2={true} feature="EU hosting" description="Access and store your data in the EU" />
  <ComparisonRow column1={true} column2={true} feature="Open source" description="View code, comment on roadmap, and contribute" />
  <ComparisonRow column1={true} column2={false} feature="Free collaboration" description="Add unlimited teammates to your project" />
  <ComparisonRow column1={true} column2={false} feature="Heatmaps" description="Visualize where users click in your app or website" />
  <ComparisonRow column1={true} column2={false} feature="Session replay" description="Watch real users use your product, diagnose bugs" />

>   <ComparisonRow column1={true} column2={false} feature="A/B testing" description="Run tests on new features, optimize conversion funnels" />

Reviewer: ```suggestion
  <ComparisonRow column1={true} column2={false} feature="Experiments" description="Run A/B/n tests on new features, optimize conversion funnels" />
```

  <ComparisonRow column1={true} column2={true} feature="Bot blocking" description="Block scrapers, crawlers, and other unwanted traffic from stats" />
  <ComparisonRow column1={true} column2={true} feature="Reverse proxy" description="Avoid tracking blockers and capture more data" />
  <ComparisonRow column1={true} column2={false} feature="HIPAA-ready" description="Can be compliant with HIPAA" />
  <ComparisonRow column1={true} column2={true} feature="GDPR-ready" description="Can be compliant with GDPR" />
  <ComparisonRow column1={true} column2={true} feature="Data anonymization" description="Options to anonymize user data" />
  <ComparisonRow column1={true} column2={false} feature="SOC 2" description="SOC 2 security certification" />
  <ComparisonRow column1={true} column2={true} feature="2FA" description="Enforce login with two-factor authentication" />
  <ComparisonRow column1="Enterprise" column2={false} feature="SAML/SSO" description="Use SAML or single sign-on authentication" />
</ComparisonTable>

## Frequently asked questions

### How much do PostHog and Plausible cost?

> Plausible is a monthly subscription based on usage. Plans start at $9/m for 10k events. Their "Business" plan unlocks more team members, custom properties, funnels, and more for a minimum of $10/m more.

Reviewer: ```suggestion
Plausible is a monthly subscription based on usage. Plans start at $9/m for 10k events. Its Business plan unlocks more team members, custom properties, funnels, and more for a minimum of $10/m more.
```
